### ISSUE TITLE
Add notification of nightly test failures

### ISSUE THREAD

**Issue by grayside** [GoogleCloudPlatform/emblem#414]
## Proposal

Add notification of nightly test failures. Email is okay, but preference would be to open issues, and not open a new issue if an existing test failure issue is open.

## Problems this will solve

Ensuring test failures are in the team's work queue and dashboards without daily manual review.

**Comment by ace-n** [comment]
**TODO:** @ace-n to collaborate with @glasnt and see what the best approach is.

_(Context: currently, our CI processes run in **Cloud Build**.)_

**Comment by ace-n** [comment]
**Note to Googlers:** this is blocked on b/229753500

**Comment by ace-n** [comment]
There are currently **three** nightly tests running in our Emblem setup, per our [testing documentation](/docs/testing.md):
1. Content API
2. Installation Test
3. Website

@rogerthatdev and I will create separate PRs to enable **build failure notifications** for each.

**Comment by ace-n** [comment]
@rogerthatdev and I met today

**Takeaway**
We should document the "plugin model" used by the Installation tests.

**Next steps**
- [Issue Comment] Identify the TF resources that we need based on [this tutorial](https://cloud.google.com/build/docs/configuring-notifications/create-notifier#configuring_notifications)
- [PR #1] Put those in a PR in their own plugin
- [PR #2] Add Dockerfile contents
- [PR #3] Get everything working end-to-end

**Comment by ace-n** [comment]
Assigning @rogerthatdev since we've synced on this.

_(Roger - as always, feel free to ask me any questions you might have! 🙂 )_

**Comment by ace-n** [comment]
Downgrading to P1, since this is not required for our next major milestone ("Public Preview")
